wmn-data.json

```diff
@@ -3078,6 +3078,18 @@
         "cat" : "health",
         "valid" : true
        },
+       {
+        "name" : "my_instants",
+        "uri_check" : "https://www.myinstants.com/en/profile/{account}/",
+        "post_body" : "",
+        "e_code" : 200,
+        "e_string" : " | Myinstants</title>",
+        "m_string" : "Page not found",
+        "m_code" : 404,
+        "known" : ["daniel01", "dave"],
+        "cate" : "music",
```

Comment:
This need to be 'cat' instead of 'cate' 🐱

Reply:
@WebBreacher Thanks for the heads up! Will correct now.